libsakura/CMakeLists.txt

```
@@ -77,19 +77,27 @@ endif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
 
 message(STATUS "CMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}")
 
+option(ENABLE_TEST "Enable tests using Google Test" OFF)
```

@cquike, thank you for submitting very useful updates. I would like to suggest one thing. Could you set the default value for `ENABLE_TEST` to `ON`? That is because I want to enable it for standalone libsakura users. 

```suggestion
option(ENABLE_TEST "Enable tests using Google Test" ON)
```

---

Sure,
I have pushed the changes.

---

I have also rebased the changes on top of main, to avoid the conflict